clang/runtime/dpct-rt/include/dpct/shmem_utils.hpp

```diff
@@ -0,0 +1,45 @@
+//==--------------- shmem_utils.hpp ---------------*- C++ -*----------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef __DPCT_SHMEM_UTILS_HPP__
+#define __DPCT_SHMEM_UTILS_HPP__
```

Comment:
no header file dependency for this header?

Reply:
nvshmem header file is migrated to ishmem, so no new dependency is added in this utils header file